tests/Goteo/Repository/InvestOriginRepositoryTest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Goteo\Repository\Tests;
+
+use Goteo\Application\Exception\ModelNotFoundException;
+use Goteo\Entity\Invest\InvestOrigin;
+use Goteo\TestCase;
+use Goteo\Repository\InvestOriginRepository;
+
+class InvestOriginRepositoryTest extends TestCase
+{
+
+    private InvestOriginRepository $repository;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+        $this->repository = new InvestOriginRepository();
+    }
+
+    public function testExceptionOnGetByInvestId(): void {
+        $id = 1;
+        $this->expectException(ModelNotFoundException::class);
+        $this->expectExceptionMessage("InvestOrigin with invest_id {$id} not found");
+
+        $this->repository->getByInvestId($id);
+
```

Comment:
Could this line be removed please?

Reply:
I think not... This test expects the ModelNotFound exception and the message written, so it has to execute the code that actually throws the esception.

As far as i understand, this is the correct way of doing this test.

Reply:
I meant the empty line, line 27.

Reply:
Ah ok! I though you meant the call to getByInvestId!